package/src/components/Reply/Reply.tsx

```diff
@@ -107,7 +107,9 @@ const ReplyWithContext = <
   const lastAttachment = quotedMessage.attachments?.slice(-1)[0] as Attachment<StreamChatGenerics>;
 
   const messageType = lastAttachment
-    ? lastAttachment.type === 'file' || lastAttachment.type === 'audio'
+    ? lastAttachment.type === 'file' ||
+      lastAttachment.type === 'audio' ||
+      lastAttachment.type === 'video'
       ? 'file'
       : lastAttachment.type === 'image' &&
         !lastAttachment.title_link &&
```

Comment:
While we're changing this one, it would be nice to refactor this massive nested ternary